packages/react/src/components/Popover/index.ts

```diff
@@ -1,2 +1,16 @@
-export type { PopoverProps } from './Popover';
-export { Popover } from './Popover';
+import type { PopoverProps } from './Popover';
+import { Popover as PopoverRoot } from './Popover';
+import type { PopoverContentType } from './PopoverContent';
+import { PopoverContent } from './PopoverContent';
+
+type PopoverComponent = React.ForwardRefExoticComponent<
+  PopoverProps & React.RefAttributes<HTMLDivElement>
+> & {
+  Content: PopoverContentType;

```

Comment:
Suggest just you using `typeof PopoverContent` and `typeof PopoverRoot` here instead of copying Reacts definition. This will be more future proof incase we refactor something in these components or React themselves change the return type for `forwardRef`